lib/elixir/lib/range.ex

```diff
@@ -47,6 +48,12 @@ defmodule Range do
       false
 
   """
+  # @spec range?(term) :: boolean
+  @spec range?(arg) :: false when arg: %Range{first: nil, last: nil}
+  @spec range?(arg) :: true when arg: %Range{}
+  @spec range?(term) :: false 
```

Comment:
Is this the right way to define the specs for this function? i'm no too sure about the last line, if it should be false, or boolean.